vcr/issuer/issuer.go

```diff
@@ -292,6 +325,38 @@ func (i issuer) Revoke(ctx context.Context, credentialID ssi.URI) (*credential.R
 	return revocation, nil
 }
 
+// revokeStatusList revokes a credential through its credential status
+func (i issuer) revokeStatusList(ctx context.Context, credentialID ssi.URI) error {
+	// TODO: If found it was issued by this node. Should this include a test to verify we still have the key and can publish the StatusList credential?
```

Comment:
I don't think you need to check whether there's a private key for the DID. Since it's a did:web, a new private key can always be added